core/templates/dev/head/components/question-directives/question-player/question-player.directive.ts

```diff
@@ -154,12 +232,93 @@ oppia.directive('questionPlayer', [
             return ctrl.totalQuestions;
           };
 
+          var createScorePerSkillMapping = function() {
+            var scorePerSkillMapping = {};
+            if (questionPlayerConfig.skillList) {
+              for (var i = 0; i < questionPlayerConfig.skillList.length; i++) {
+                var skillId = questionPlayerConfig.skillList[i];
+                var description = questionPlayerConfig.skillDescriptions[i];
+                scorePerSkillMapping[skillId] = {
+                  description: description,
+                  score: 0.0
+                };
+              }
+            }
+            return scorePerSkillMapping;
+          };
+
+          var calculateScores = function(questionStateData) {
+            var scorePerSkillMapping = createScorePerSkillMapping();
+            $scope.resultsLoaded = false;
+            var totalScore = 0.0;
+            var minScore = Number.MAX_VALUE;
```

Comment:
This doesn't seem to be used?

Reply:
It will be used for Boost Score in a follow up but I will remove it for now